docs/intro.md

```diff
@@ -11,3 +11,30 @@ Auto-instrumentation means it will automatically detect the libraries you use an
 OpenTelemetry is a set of tools, APIs and SDKs that provide standardized approach to telemetry including: instrumenting, generating, collecting and exporting telemetry data.
 
 Mesmer is designed as an extension for the OpenTelemetry Java agent. This way you will benefit from both the general Java APIs metrics that are still useful in Scala (like JDBC or Netty) and the specific Scala instrumentation provided by Mesmer (like Akka or ZIO).
+
+# Links to get started:
+
+## ByteBuddy
```

Comment:
Not super important: all the links below could use `[Link makrdown support](HTTP://foo.bar)`

Reply:
@lgajowy, refactored in this commit:
https://github.com/ScalaConsultants/mesmer/pull/470/commits